docs/root/start/_include/envoy-dynamic-demo.yaml

```diff
@@ -0,0 +1,56 @@
+node:
+  cluster: test-cluster
+  id: test-id
+
+dynamic_resources:
+  ads_config:
+    api_type: GRPC
+    transport_api_version: V3
+    grpc_services:
+    - envoy_grpc:
+        cluster_name: xds_cluster
+    set_node_on_first_message_only: false
+  cds_config:
+    resource_api_version: V3
+    ads: {}
+  lds_config:
+    resource_api_version: V3
+    ads: {}
+
+static_resources:
+  clusters:
+  - connect_timeout: 1s
+    type: strict_dns
+    load_assignment:
+      cluster_name: xds_cluster
+      endpoints:
+      - lb_endpoints:
+        - endpoint:
+            address:
+              socket_address:
+                address: go-control-plane
+                port_value: 18000
+    http2_protocol_options: {}
+    name: xds_cluster
+
+layered_runtime:
+  layers:
+    - name: runtime-0
+      rtds_layer:
+        rtds_config:
+          resource_api_version: V3
+          ads: {}
+        name: runtime-0
+    - name: runtime-1
```

Comment:
im wondering if there is any need to have a second runtime layer like this - and if so - ill add to docs why

Reply:
i removed it and the example still seems to work ok